lib/Serge/Engine/Plugin/parse_rc.pm

```diff
@@ -93,13 +93,21 @@ sub parse {
 
         if ($orig_str) {
             my $str = $orig_str;
+
+            my $wide = $str =~ /^L/;
+            $str =~ s/^L?"(.*)"$/\1/g;
+            if ($wide) {
+                $str =~ s/\\x([0-9a-fA-F]{4})/chr(hex($1))/ge;
```

Comment:
Shouldn't we also replace the hex for characters on non-wide ASCII strings? From the docs you referred:
> The following example shows strings with both ASCII and Unicode. Note that strings without the initial "L" use the 2-digit escape format:

Reply:
I have found no good documentation on how to interpret 2-digit escapes in RC files. I try to avoid them as much as possible in my code. I believe they are interpreted using either the current system's code page or the matching `#pragma code_page(...)`, either of which would require more Perl than I am comfortable with. 

This article gives a little bit more details: https://devblogs.microsoft.com/oldnewthing/20190607-00/?p=102569